### PR TITLE
Migrate pricing page

### PR DESCRIPTION
**Pull request by shawyu-okta**
## Description:
- **What's changed?**
  * Adds /pricing page migrated from existing developer.okta.com/pricing
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-349866](https://oktainc.atlassian.net/browse/OKTA-349866)

### CODE REVIEW

**Comment by francoislehoux-okta** [comment]
Is this new page not reachable when not building with the redesign flag?

**Comment by shawyu-okta** [comment]
> Is this new page not reachable when not building with the redesign flag?

@francoislehoux-okta I believe since the `/pricing` route is directed to a different app by the parent proxy/cloudfront in AWS it shouldn't be visible. When we direct all routes to this Vuepress app, it should just start working. I think we have to understand the current setup to ascertain for sure. cc @haishengwu-okta

**Review by francoislehoux-okta — APPROVED** [review]
LGTM as long as we're sure the routing will handle this once this is merged

**Review by volodymyrmakarchuk-okta — DISMISSED** [review]

File: packages/@okta/vuepress-theme-prose/index.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = (options, ctx) => {
       // Set to true to enable the redesign in development
       const redesign = process.env.REDESIGN || false;
       // Respect the redesign flag unless this is a production build
-      $page.redesign = !ctx.isProd && redesign;
+      $page.redesign = true;//!ctx.isProd && redesign;
```
[the remark this answers is not in the document]
Comment (volodymyrmakarchuk-okta) [reply packages/@okta/vuepress-theme-prose/index.js:19]: ah that's for netlify

[the remark this answers is not in the document]
Comment (shawyu-okta) [reply packages/@okta/vuepress-theme-prose/index.js:19]: Oh yeah this is inside a commit I am going to revert before merge - wanted to try having it ON in a test deployment to show off the footer.

File: packages/@okta/vuepress-theme-prose/components/Pricing.vue [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+                    <div class="pricing-card-column">
+                      {{details.name}}
+                      <ul v-if="details.bullets">
+                        <li v-for="bullet in details.bullets">
+                          {{bullet}}
+                        </li>
+                      </ul>
+                    </div>
+                    <template v-for="edition in $page.frontmatter.editions">
+                      <div class="pricing-card-column" v-if="typeof edition[feature] === 'object'">
```
[the remark this answers is not in the document]
Comment (shawyu-okta) [reply packages/@okta/vuepress-theme-prose/components/Pricing.vue:115]: Shouldn't be possible for it to be `null`. `undefined` is possible I guess but we aren't planning on scaling this at all so as long as we don't touch the `.md` file that this uses for template data, there's not much risk.

**Pull request merged**